src/display/annotation_layer.js

```diff
@@ -2909,13 +2910,24 @@ class HighlightAnnotationElement extends AnnotationElement {
   }
 
   render() {
-    if (!this.data.popupRef && this.hasPopupData) {
+    const {
+      data: { highlightedText, popupRef },
+    } = this;
+
```

Comment:
Let's remove this newline given that the other three classes below don't have it as well (alternatively, add it there too).